app/components/InventoryOverview/index.js

```diff
@@ -225,22 +225,17 @@ const InventoryOverview = ({ navigation }) => {
           features: featureList,
         };
         let fileName = `Tree Mapper GeoJSON ${inventory.inventory_id}.json`;
-        let path = `${
-          Platform.OS == 'ios' ? RNFetchBlob.fs.dirs.DocumentDir : RNFetchBlob.fs.dirs.DownloadDir
-        }/${fileName}`;
+        let path = `${RNFetchBlob.fs.dirs.DocumentDir}/${fileName}`;
```

Comment:
Android - This will store the file inside the package folder under Android/data/org.pftp.treemapper/files/ and the files in this folder are not visible to user. Either it can be stored in a seperate folder of tree mapper or in downloads folder.

Reply:
@ankitecd yes, I know. But using RNFetchBlob.fs.dirs.DownloadDir you get a permission error in Android 10!

Reply:
ohh okay got it.

Reply:
changed to a sharing functionality